<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchGroupDialog</class>
 <widget class="QDialog" name="BatchGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量添加用户组</string>
  </property>
  <widget class="QPushButton" name="ptnDelete">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnModify">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnAdd">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnSave">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnClose">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="groupTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>129</y>
     <width>620</width>
     <height>470</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>组编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>组名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>组权限</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>组名称*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtGroupName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>组权限*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spbGroupRight">
      <property name="maximum">
       <number>1048576</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>备   注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtGroupComments"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>组编号*：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtGroupNo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>ptnDelete</zorder>
  <zorder>ptnModify</zorder>
  <zorder>ptnAdd</zorder>
  <zorder>ptnSave</zorder>
  <zorder>ptnClose</zorder>
  <zorder>groupTableWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
